scaling_performance/optimizing_storage.adoc

```diff
@@ -126,5 +126,21 @@ Overlay is also supported for Docker use cases as of Red Hat Enterprise Linux
 7.2, and provides faster start up time and page cache sharing, which can
 potentially improve density by reducing overall memory utilization.
 
+[[benefits-of-using-the-overlay-graph-driver]]
+== Benefits of Using the Overlay Graph Driver with SELinux
 
+Now that Overlay is in the upstream kernel and support for SELinux is extended,
+it is recommended that you keep Device Mapper as the default in Red Hat
+Enterprise Linux, but change the default graph driver to Overlay2 in Fedora 26.
 
+As of Red Hat Enterprise Linux 7.3, the Overlay2 back-end is supported. The
+Overlay file system has several advantages over device mapper (most importantly,
+page cache sharing among containers). Support for the Overlay file system was
+added to Red Hat Enterprise Linux with salient caveats: It was not POSIX
+compliant and the use of Overlay was, at the time, incompatible with SELinux (a
+key security and isolation technology). The density improvements gained by page
+cache sharing are very important for certain environments where there is
+significant overlap in base image content.
+
+Red Hat testing found Overlay2 to be very stable, and that stability is
+furthered with the addition of SELinux support.
```

Comment:
How about this:

The default docker storage configuration on RHEL remains device-mapper.  As of Red Hat Enterprise Linux 7.2, overlay became a supported graph driver.  And, as of Red Hat Enterprise Linux 7.4, SELinux and the overlay2 graph driver became a supported combination.

The main advantage of the overlay filesystem is Linux page cache sharing among containers on the same node.  This attribute of overlay leads to reduced I/O during container startup (and thus faster container startup time by several hundred milliseconds), as well as reduced memory usage when similar images are running on a node.  Both side-effects lend themselves well towards many environments, especially those with the goal of optimizing for density and/or that have high container churn rate (such as a build farm), or those that have significant overlap in image content.

Page cache sharing is not possible with device-mapper due to how allocating thin-provisioned devices on a per-container basis.